src/EditorFeatures/Test/Diagnostics/DiagnosticsClassificationTaggerProviderTests.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.CodeAnalysis.Diagnostics;
```

Comment:
i'm curious why this wasn't caught before.  were the tests incomplete?  incorrect?